randovania/data/json_data/prime1.txt

```diff
@@ -1580,411 +1583,799 @@ Tallon Overworld
 Root Tunnel
 Asset id: 1078461657
-> Dock to Root Cave; Heals? False
-  * Normal Door to Root Cave/Dock to Root Tunnel
+> Dock to Root Cave; Heals? False; Spawn Point
+  * Missile Blast Shield to Root Cave/Dock to Root Tunnel
+  > Dock to Tallon Canyon
+      Trivial
 
 > Dock to Tallon Canyon; Heals? False
   * Normal Door to Tallon Canyon/Dock to Root Tunnel
+  > Dock to Root Cave
+      Trivial
 
 ----------------
 Temple Lobby
 Asset id: 591880894
-> Dock to Artifact Temple; Heals? False
+> Dock to Artifact Temple; Heals? False; Spawn Point
   * Normal Door to Artifact Temple/Dock to Temple Lobby
+  > Dock to Temple Security Station
+      Trivial
 
 > Dock to Temple Security Station; Heals? False
-  * Normal Door to Temple Security Station/Dock to Temple Lobby
+  * Missile Blast Shield to Temple Security Station/Dock to Temple Lobby
+  > Dock to Artifact Temple
+      Trivial
 
 ----------------
 Frigate Access Tunnel
 Asset id: 3138751614
-> Dock to Main Ventilation Shaft Section C; Heals? False
+> Dock to Main Ventilation Shaft Section C; Heals? False; Spawn Point
   * Normal Door to Main Ventilation Shaft Section C/Dock to Frigate Access Tunnel
+  > Dock to Frigate Crash Site
+      Trivial
 
 > Dock to Frigate Crash Site; Heals? False
-  * Normal Door to Frigate Crash Site/Dock to Frigate Access Tunnel
+  * Ice Door to Frigate Crash Site/Dock to Frigate Access Tunnel
+  > Dock to Main Ventilation Shaft Section C
+      Trivial
 
 ----------------
 Overgrown Cavern
 Asset id: 3466748899
-> Dock to Frigate Crash Site; Heals? False
-  * Normal Door to Frigate Crash Site/Dock to Overgrown Cavern
+> Dock to Frigate Crash Site; Heals? False; Spawn Point
+  * Ice Door to Frigate Crash Site/Dock to Overgrown Cavern
+  > Dock to Transport Tunnel C
+      Morph Ball
 
 > Dock to Transport Tunnel C; Heals? False
-  * Normal Door to Transport Tunnel C/Dock to Overgrown Cavern
+  * Ice Door to Transport Tunnel C/Dock to Overgrown Cavern
+  > Dock to Frigate Crash Site
+      Morph Ball
+  > Pickup (Missile Expansion)
+      Morph Ball
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 61
+  > Dock to Transport Tunnel C
+      Morph Ball
 
 ----------------
 Transport to Chozo Ruins West
 Asset id: 295707720
-> Dock to Transport Tunnel A; Heals? False
+> Dock to Transport Tunnel A; Heals? False; Spawn Point
   * Normal Door to Transport Tunnel A/Dock to Transport to Chozo Ruins West
+  > Elevator to Chozo Ruins - Transport to Tallon Overworld North
+      Scan Visor
+
+> Elevator to Chozo Ruins - Transport to Tallon Overworld North; Heals? False
+  * Teleporter to Chozo Ruins/Transport to Tallon Overworld North
+  > Dock to Transport Tunnel A
+      Trivial
 
 ----------------
 Root Cave
 Asset id: 3180103205
-> Dock to Transport Tunnel B; Heals? False
+> Dock to Transport Tunnel B; Heals? False; Spawn Point
   * Normal Door to Transport Tunnel B/Dock to Root Cave
+  > Dock to Root Tunnel
+      Trivial
+  > Before Pickup
+      Grapple Beam
```

Comment:
Before Pickup isn't necessary here, as you can just go from Root Tunnel door instead